test/internal/modules/waiting/controller.js

```diff
@@ -141,3 +179,84 @@ experiment('getWaiting', () => {
     });
   });
 });
+
+// ##### billing event tests ###########
```

Comment:
This could all be wrapped in an experiment like

`experiment('internal/modules/waiting/controller', () => {`

Reply:
done